realm/realm-library/src/main/java/io/realm/RealmConfiguration.java

```diff
@@ -435,7 +435,8 @@ public String toString() {
      * @return {@code true} if RxJava dependency exist, {@code false} otherwise.
      */
     @SuppressWarnings("LiteralClassName")
-    static synchronized boolean isRxJavaAvailable() {
+    // FIXME Review public exposure - Needed from SyncConfiguration.Builder can we expose this?
+    public static synchronized boolean isRxJavaAvailable() {
```

Comment:
Probably doesn't hurt, although it does bloat the public API for very little value.

Reply:
Yep. Will move it to internal util.